common/params.cc

```diff
@@ -156,6 +156,7 @@ std::unordered_map<std::string, uint32_t> keys = {
     {"LiveTorqueParameters", PERSISTENT | DONT_LOG},
     {"LongitudinalPersonality", PERSISTENT},
     {"NavDestination", CLEAR_ON_MANAGER_START | CLEAR_ON_OFFROAD_TRANSITION},
+    {"NavDestinations", PERSISTENT},
```

Comment:
let's call this `NavPastDestinations`, this one looks too similar to NavDestination